packages/base/package.json

```diff
@@ -15,6 +15,9 @@
   "author": {
     "name": "JupyterGIS contributors"
   },
+  "jupyterlab": {
+    "webpackConfig": "./webpack.config.js"
+  },
```

Comment:
The `base` package is not a labextension, this won't be used

```suggestion
```

Reply:
That makes tons of sense :laughing: Thanks!

I don't know if there's a way to address this problem without adding another tool to the toolchain, and I don't like the idea of making our build process more complicated.

Let's just punt on this for now :( Importing from `../../../../foo` is really hard to read but it's not the end of the world. SWC seems to be able to resolve the aliases at compile time if we decide it's worth adding a tool to our toolchain.